largest_sum_contiguous_subarray/LargestSumContiguousSubarray.java

```diff
@@ -0,0 +1,38 @@
+/* 
+   *   Implementation of famous dynamic programming problem
+   *   Largest Sum Contiguous Subarray
+   *   Kadane Algorithm 
+   *   Time complexity  O(n)
+*/
+
+class LargestSumContiguousSubarray {
```

Comment:
We can remove certain checks that seem problematic. Removing the whole check system wouldn't be good.